control-plane/catalog/to-k8s/sink.go

```diff
@@ -19,6 +21,25 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
+var SyncToK8sCounters = []prometheus.CounterDefinition{
+	{
+		Name: []string{"consul", "sync_catalog", "to_k8s"},
```

Comment:
similar idea here, can we move these slices to variables and reuse them through the code and should we include "register" here as we use "deregister" for the opposite?